radix-engine-tests/tests/misc/typed_manifest_invocation.rs

```diff
@@ -0,0 +1,189 @@
+        FunctionTable::construct_from_typed_invocation_function_table();
+
+    /* Assert */
+    // We're taking the blueprint names in the typed invocations enum as the canonical set of
+    // blueprints since we don't want to check that we support all blueprints, just all functions.
+    for (blueprint_name, typed_native_invocation_blueprint_function_table) in
+        typed_manifest_invocations_function_table.0.into_iter()
+    {
+        let Some(canonical_blueprint_function_table) =
+            ledger_function_table.0.get(&blueprint_name).cloned()
+        else {
+            panic!(
+                "Blueprint \"{blueprint_name:?}\" is in the typed manifest enum but not on ledger"
+            );
+        };
+
+        for function_type in [
+            FunctionType::Function,
+            FunctionType::Method,
+            FunctionType::DirectMethod,
+        ] {
+            let typed_invocations_blueprint_function_type_functions =
+                typed_native_invocation_blueprint_function_table
+                    .0
+                    .get(&function_type);
+            let canonical_blueprint_function_type_functions =
+                canonical_blueprint_function_table.0.get(&function_type);
+
+            match (
+                typed_invocations_blueprint_function_type_functions,
+                canonical_blueprint_function_type_functions,
+            ) {
+                (
+                    Some(typed_invocations_blueprint_function_type_functions),
+                    Some(canonical_blueprint_function_type_functions),
+                ) => {
+                    let functions_in_typed_invocation_but_not_in_ledger =
+                        typed_invocations_blueprint_function_type_functions
+                            .difference(canonical_blueprint_function_type_functions)
+                            .collect::<HashSet<_>>();
+                    let functions_in_ledger_but_not_in_typed_invocation =
+                        canonical_blueprint_function_type_functions
+                            .difference(typed_invocations_blueprint_function_type_functions)
+                            .collect::<HashSet<_>>();
+
+                    if !functions_in_typed_invocation_but_not_in_ledger.is_empty() {
+                        panic!("For blueprint \"{blueprint_name:?}\" and function type \"{function_type:?}\" the following set of functions are defined in the typed invocation enum but are not present on ledger: {functions_in_typed_invocation_but_not_in_ledger:?}")
+                    }
+                    if !functions_in_ledger_but_not_in_typed_invocation.is_empty() {
+                        panic!("For blueprint \"{blueprint_name:?}\" and function type \"{function_type:?}\" the following set of functions are defined in ledger but are not present in the typed invocations enum: {functions_in_ledger_but_not_in_typed_invocation:?}")
+                    }
+                }
```

Comment:
It feels like we shouldn't stop here, and we should actually check that they also have the same schema if possible? :)

We can do this by storing a `ScryptoSingleTypeSchema` from each, and then:

```rust
compare_single_type_schemas(
    SchemaComparisonSettings::require_equality(),
    blueprint_schema,
    typed_schema,
).assert_valid("blueprint definition", "typing definition");
```